### DISCUSSION TITLE
Subtitles

### DISCUSSION THREAD

**Discussion by adamesq** [rrirower/DP2SceneGraphWizard#44] [General]
Anyone have any luck with subtitles? My .srt has the exact same filename. It's s showing in my feed. It's even in the same directory as the video file. Feed looks fine. I've run the subtitle through a validator. When I open the video in VLC, VLC sees and loads the .srt file. 🤔 

Is this issue on Roku's side?

**Comment by rrirower** [comment]
Keep in mind, the code produced from the wizard is not complete. It's evolving every day. As such, subtitles and other things have not been implemented yet.

**Reply by adamesq** [reply]
GOT IT! Thank you. Thought I messed something up.

**Reply by rrirower** [reply]
Captions are implemented in alpha-8.
